test/ui/selenium/LoginTests.js

```diff
@@ -0,0 +1,252 @@
+const { Assertion } = require('chai');
+const { assert } = require('chai');
+const {Builder, By, TouchSequence} = require('selenium-webdriver');
+const HomePage = require('../POM/HomePage');
+const LoginPage = require('../POM/LoginPage');
+const readline = require('readline');
+const { TsJestCompiler } = require('ts-jest');
+const email = process.env.LOGIN_EMAIL;
+const password = process.env.LOGIN_PASSWORD;
+const adminEmail = process.env.LOGIN_EMAIL_ADMIN;
+const adminPassword = process.env.LOGIN_PASSWORD_ADMIN;
+require('dotenv').config();
```

Comment:
Is this code the wrong way round? Why is the config being required after already reading from the .env?